style-guide/src/pages/handover-process.js

```diff
@@ -1,17 +1,85 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import Layout from '../components/layout/layout';
-import styles from '../styles/index.module.scss';
+import styles from '../styles/handover.module.scss';
 import SectionHeader from '../components/section-header';
+import copy from '../../static/handover-process';
+import banner from '../assets/handoverBanner.png';
+import amazeDesigners from '../assets/amazeDesigners.png';
+import PhaseOne from "../components/handover-process/PhaseOne";
+import PhaseTwo from "../components/handover-process/PhaseTwo";
+import PhaseThree from "../components/handover-process/PhaseThree";
+import PhaseFour from "../components/handover-process/PhaseFour";
+import PhaseFive from "../components/handover-process/PhaseFive";
+import storyMapping from "../assets/storyMapping.png";
+
+const navItems = ['phase1', 'phase2', 'phase3', 'phase4', 'phase5'];
 
 export default () => {
+  const [selectedPhase, setSelectedPhase] = useState(null);
+
+  useEffect(() => {
+    const phaseNumber = window.location.hash.replace("#", '');
+    setSelectedPhase(phaseNumber);
+  }, []);
+
+  useEffect(() => {
+    scrollTo(selectedPhase);
+  }, [selectedPhase]);
+
+  function scrollTo(phaseNumber) {
+    if (phaseNumber) {
+      const elem = document.getElementById(phaseNumber);
+      elem.scrollIntoView({behavior: 'smooth', block: 'start', inline: "nearest"})
+    }
+  }
+
+  function addHashRoute(phaseNumber) {
+    if (window.history.pushState) {
+      window.history.pushState(null, null, `#${phaseNumber}`);
+    } else {
+      window.location.hash = `#${phaseNumber}`;
+    }
+
+    setSelectedPhase(phaseNumber);
+  }
```

Comment:
Can these functions be moved outside of the handover-processes? They'll be redefined on each render otherwise.